pkg/controller/plan/migration.go

```diff
@@ -757,6 +757,11 @@ func (r *Migration) execute(vm *plan.VMStatus) (err error) {
 			err = liberr.Wrap(err)
 			return
 		}
+		err = r.deleteImporterPods(vm)
+		if err != nil {
+			err = liberr.Wrap(err)
+			return
```

Comment:
also here, and when handling error for the calls above, why do we return?
when we return an error the reconcile fails and we'll try again, right? in that case, I'd proceed with try to remove all leftovers (data volumes, init pods, importer pods) and return an error if any of them failed

Reply:
i changed it a bit.